### ISSUE TITLE
ko apply -f config/ failed with the following error, for the pod knative-serving-operator-755f4687d7-chtjn

### ISSUE THREAD

**Issue by houshengbo** [knative/serving-operator#32]
```
{"level":"info","ts":1561170741.6739466,"logger":"cmd","msg":"Go Version: go1.12.5"}
{"level":"info","ts":1561170741.6739893,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
{"level":"info","ts":1561170741.673994,"logger":"cmd","msg":"Version of operator-sdk: v0.8.0"}
{"level":"info","ts":1561170741.674168,"logger":"leader","msg":"Trying to become the leader."}
{"level":"info","ts":1561170741.8183033,"logger":"leader","msg":"Found existing lock with my name. I was likely restarted."}
{"level":"info","ts":1561170741.8183417,"logger":"leader","msg":"Continuing as the leader."}
{"level":"info","ts":1561170742.1300085,"logger":"cmd","msg":"Registering Components."}
{"level":"info","ts":1561170742.130198,"logger":"manifestival","msg":"Reading file","name":"/var/run/ko/knative-serving-0.6.0.yaml"}
{"level":"info","ts":1561170742.1389608,"logger":"manifestival","msg":"Reading file","name":"deploy/crds/serving_v1alpha1_knativeserving_cr.yaml"}
{"level":"error","ts":1561170742.1390326,"logger":"cmd","msg":"","error":"stat deploy/crds/serving_v1alpha1_knativeserving_cr.yaml: no such file or directory","stacktrace":"github.com/knative/serving-operator/vendor/github.com/go-logr/zapr.(*zapLogger).Error\n\t/Users/vincent/go/src/github.com/knative/serving-operator/vendor/github.com/go-logr/zapr/zapr.go:128\nmain.main\n\t/Users/vincent/go/src/github.com/knative/serving-operator/cmd/manager/main.go:123\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:200"}
```

**Comment by trshafer** [comment]
@houshengbo is this now resolved by merging in #23 and #37?

**Comment by houshengbo** [comment]
@trshafer Thx,
this issue was closed by the PR #23.
